server/historian/packages/historian-base/src/routes/git/blobs.ts

```diff
@@ -93,6 +93,7 @@ export function create(
 			...tenantThrottleOptions,
 			throttleIdPrefix: "ping",
 		}),
+		// eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
I'm fine with the comment but in fact I think this route might not work correctly today? AFAIK express 4.x does not support async route handlers. Just as a note.

Reply:
I can add TODO notes to these as well. I didn't want to make any functional changes, and the immediate goal is to prevent further violations. But I'm happy to add notes.

Reply:
There were something on the order of 10 or so violations of this rule. So, an investigation is definitely warranted.

Reply:
Actually, scanning the code for the disable comment, there are many more than those that I added. This seems to be a larger issue. I might just leave the disables as is for now.

@znewton This might be interesting for someone on your team to look at.